packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -116,14 +123,18 @@ module.exports = {
       endsWith: "string/ends-with",
       fromCodePoint: "string/from-code-point",
       includes: "string/includes",
-      padLeft: "string/pad-left",
-      padRight: "string/pad-right",
+      padLeft: "string/pad-start", // deprecated
+      padRight: "string/pad-end", // deprecated
```

Comment:
Yep, I forgot about it. The special case for it in the build script makes sense. Any other ideas how to fix it?